_test/pages_api_test.rb

```diff
@@ -0,0 +1,59 @@
+# 18F Hub - Docs & connections between team members, projects, and skill sets
+#
+# Written in 2015 by Aidan Feldman (aidan.feldman@gsa.gov)
+# on behalf of the 18F team, part of the US General Services Administration:
+# https://18f.gsa.gov/
+#
+# To the extent possible under law, the author(s) have dedicated all copyright
+# and related and neighboring rights to this software to the public domain
+# worldwide. This software is distributed without any warranty.
+#
+# You should have received a copy of the CC0 Public Domain Dedication along
+# with this software. If not, see
+# <https://creativecommons.org/publicdomain/zero/1.0/>.
+#
+# @author Aidan Feldman (aidan.feldman@gsa.gov)
+
+require_relative 'test_helper'
+
+require 'minitest/autorun'
+
+module Hub
+  class PagesApiTest < ::Minitest::Test
+    # before all
+    # TODO isolate the tests better
+    BUILD_DIR = File.join(Dir.pwd, '_test', 'tmp')
+    `bundle exec jekyll build --destination #{BUILD_DIR}`
```

Comment:
I added this file of tests before moving the plugin to a separate gem, so honestly wouldn't be opposed to removing this entire file. I am [doing the same thing in an integration test](https://github.com/18F/jekyll_pages_api/blob/6df7299eca91ad22de2fdad837b3e6efc954f50e/spec/integration_spec.rb#L27) in jekyll_pages_api, though there are now additional unit tests in there.